test/system/src/test/java/io/pravega/test/system/MultiReaderTxnWriterWithFailoverTest.java

```diff
@@ -305,7 +303,30 @@ public void multiReaderTxnWriterWithFailOverTest() throws Exception {
         log.info("Test {} succeeds ", "MultiReaderWriterTxnWithFailOver");
     }
 
-    private void performFailoverTest() throws InterruptedException {
+    private void closeReadersAndWriters() {
+        log.info("Closing writers");
+        writerList.forEach(writer -> {
+            try {
+                writer.close();
+            } catch (RetriesExhaustedException e) {
+                log.warn("Unable to close the client: ", e);
```

Comment:
@tkaitchuck here we are closing the readers and writers, which happens at the end of the test. The goal of the test is not to test whether the client is closing correct, so it begs the question of whether we should fall the whole test case here or log and proceed. My sense is that we should be testing whether the client is closing correctly elsewhere, but I'm fine with being strict and failing if it doesn't close correctly.